tests/integration/_cases/org_tokens/url-match.trycmd

```diff
@@ -1,11 +1,9 @@
 ```
-$ sentry-cli --auth-token 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 --url http://localhost:8000 sourcemaps upload
+$ sentry-cli --auth-token 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 --url http://localhost:8000 sourcemaps upload --org nomatch  test_path
 ? failed
-error: the following required arguments were not provided:
-  <PATHS>...
+error: Two different org values supplied: `sentry` (from token), `nomatch`.
 
-Usage: sentry-cli[EXE] sourcemaps upload <PATHS>...
-
-For more information, try '--help'.
+Add --log-level=[info|debug] or export SENTRY_LOG_LEVEL=[info|debug] to see more output.
```

Comment:
This test wasn't actually asserting the URL matches correctly. It appears that the `upload <paths>` check occurs before the URL checking logic is run.

Reply:
Great catch!